Source/Extensions/Blazorise.Markdown/wwwroot/markdown.js

```diff
@@ -78,6 +79,8 @@ export function initialize(dotNetObjectRef, element, elementId, options) {
         imageCSRFToken: options.imageCSRFToken,
         imageTexts: options.imageTexts,
         imageUploadFunction: (file, onSuccess, onError) => {
+            removeAllFileEntries(element);
```

Comment:
This will delete the entry in the middle of processing the uploads from mdEasy. (for multiple files the `imageUploadFunction` is called multiple times) 
And will fail with `Exception form InvokeSafeVoidAsync: There is no file with ID 1. The file list may have changed` when processing non-existing file entry.

Reply:
Fixed.